kbs/src/plugins/sample.rs

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) 2024 by Alibaba.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+//! This is a sample to implement a client plugin
+
+use actix_web::{http::Method, HttpResponse};
+use serde::Deserialize;
+
+use super::{plugin_manager::ClientPlugin, Result};
+
+#[derive(Deserialize, Clone, Debug, PartialEq)]
+pub struct SampleConfig {
+    pub item: String,
+}
```

Comment:
https://github.com/confidential-containers/trustee/pull/451#discussion_r1710574306 , it would be nice if the SampleConfig path is read from the `kbs-config.toml`?